src/utils.jl

```diff
@@ -179,10 +179,8 @@ function sha_pkg(pe::PackageEntry)
     path(pe) isa String && isdir(path(pe)) && isdir(joinpath(path(pe), "src")) ? sha2_256_dir(joinpath(path(pe), "src")) : nothing
 end
 
-function _doc(@nospecialize(object))
+function _doc(binding::Base.Docs.Binding)
```

Comment:
Any specific reason for this change? The `@nospecialize` may have been important(-ish) for performance.

Reply:
No reason – I'm just not familiar with `@nospecialize`. The function does expect a binding now. Should it be `@nospecialize(binding)` or `@nospecialize(binding::Base.Docs.Binding)`?

Reply:
The former. But this is fine as-is, since it's only ever be called with a `Binding` anyways.